imports/client/components/PuzzlePage.tsx

```diff
@@ -54,109 +50,53 @@ const FilteredChatFields: ('_id' | 'puzzle' | 'text' | 'sender' | 'timestamp')[]
 type FilteredChatMessageType = Pick<ChatMessageType, typeof FilteredChatFields[0]>
 
 const MinimumDesktopWidth = 600;
-const MinimumDesktopStackingHeight = 400; // In two column mode, allow stacking at smaller heights
-const MinimumMobileStackingHeight = 740; // Captures iPhone Plus but not iPad Mini
 const MinimumSidebarWidth = 150;
 const MinimumDocumentWidth = 375;
-const MinimumChatHeight = 96;
-
 const DefaultSidebarWidth = 300;
-const DefaultChatHeightFraction = 0.6;
 
 // PuzzlePage has some pretty unique properties:
 //
 // * It's the only page which iframes other sites.  Doing so requires that we
 //   specify the absolute size and position of the iframe, which makes us need
 //   position: fixed.
-// * There's up to four interesting pieces of content shown on this page:
-//   * Related puzzles
+// * There's up to three interesting pieces of content shown on this page:
```

Comment:
Thanks for updating all the comments in through here.  The simplification is great. :)